gno.land/cmd/gnoland/main.go

```diff
@@ -105,6 +106,13 @@ func (c *gnolandCfg) RegisterFlags(fs *flag.FlagSet) {
 		"localhost:26657",
 		"replacement for '%%REMOTE%%' in genesis",
 	)
+
+	fs.StringVar(
+		&c.rpcListenAddr,
+		"rpc.laddr",
+		"tcp://127.0.0.1:26657",
```

Comment:
This string is used 3 times in this file, and should be a constant. This constant should probably be located in a pkg, and not in the main.